### PR TITLE
[kwokctl] Support port exposure for kube-scheduler

### PR DESCRIPTION
**Pull request by nikola-jokic**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Allow access to the kube-scheduler from the host machine

#### Which issue(s) this PR fixes:

Related to #261  (kube-scheduler-port)

### CODE REVIEW

**Comment by nikola-jokic** [comment]
This change does not work on kind right now.
Kind uses in-cluster config, and not kube scheduler component.
Should we source the scheduler config and modify it to use NodePort? https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins

**Comment by wzshiming** [comment]
> This change does not work on kind right now. Kind uses in-cluster config, and not kube scheduler component. Should we source the scheduler config and modify it to use NodePort? https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins

I agree with you, let's just explain this in the flag and documentation

I'm considering whether to allow more expected behavior differences in the Kind runtime in the future, it's a bit of a workload to maintain consistency with other runtime behaviors.

File: test/kwokctl/kwokctl_restart_test.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ function test_delete_cluster() {
 
 function test_prometheus() {
   local targets
-  for ((i = 0; i < 30; i++)); do
+  for ((i = 0; i < 60; i++)); do
```
Comment (nikola-jokic) [inline test/kwokctl/kwokctl_restart_test.sh:57]: The scheduler was slow to get back up after the restart on nerdctl :relaxed:

File: test/kwokctl/kwokctl_workable_test.sh [hunk 2]
```diff
@@ -144,6 +144,28 @@ function test_etcd_port() {
...
+    minor="${minor%.*}"
+
+    local proto="https"
+    if [[  $minor -le 12 ]]; then
+        proto="http"
+    fi
+
+    result=$(curl -s -k "${proto}://127.0.0.1:10250/healthz")
+
+    if [[ ! "${result}" = "ok" ]]; then
```
Comment (wzshiming) [inline test/kwokctl/kwokctl_workable_test.sh:161]:
```suggestion
    if [[ "${result}" != "ok" ]]; then
```

**Review by wzshiming — COMMENTED** [review]
Others LGTM, and squash the commits.

**Review by wzshiming — APPROVED** [review]

**Pull request merged**